Mathlib/Analysis/SpecialFunctions/ContinuousFunctionalCalculus/Rpow/CStar.lean

```diff
@@ -0,0 +1,75 @@
+/-
+Copyright (c) 2025 Frédéric Dupuis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Frédéric Dupuis
+-/
+
+import Mathlib.Analysis.SpecialFunctions.ContinuousFunctionalCalculus.Rpow.Basic
+import Mathlib.Analysis.CStarAlgebra.ContinuousFunctionalCalculus.Order
+
+/-! # Properties of `rpow` and `sqrt` over C⋆-algebras
+
+This file collects results about `CFC.rpow`, `CFC.nnrpow` and `CFC.sqrt` that use facts that
+are specific to C⋆-algebras.
+
+## Main theorems
+
+* Various versions of `‖a ^ r‖ = ‖a‖ ^ r` and `‖CFC.sqrt a‖ = sqrt ‖a‖`.
+
+## Tags
+
+continuous functional calculus, rpow, sqrt
+-/
+
+open scoped NNReal
+
+namespace CFC
+
+section nonunital
+
+variable {A : Type*} [NonUnitalCStarAlgebra A] [PartialOrder A] [StarOrderedRing A]
```

Comment:
After the generalization in #20882, I suspect that this can be generalized to `NonUnitalIsometricContinuousFunctionalCalculus`.

Reply:
Indeed it can be generalized. I've renamed the new file from `CStar.lean` to `Isometric.lean`.